README.md

```diff
@@ -65,13 +66,15 @@ To see the test results, please check:
   
 - [Video.js](/docs/video.js#test-results)
 - [Shaka Player](/docs/shaka#test-results)
+- [THEOplayer](/docs/THEOplayer#test-results)
 
 ## How to set up your player
 
 To set up your player, follow the below instructions:
 
 - [How to use the Video.js player with Azure Media Services](https://docs.microsoft.com/azure/media-services/latest/how-to-video-js-player)
 - [How to use the Shaka player with Azure Media Services](https://docs.microsoft.com/azure/media-services/latest/how-to-shaka-player)
+- [How to use the THEOplayer with Azure Media Services](https://docs.microsoft.com/azure/media-services/latest/how-to-theo-player)
```

Comment:
We don't have this link yet. Do you want to link it to your doc for now?

Reply:
@ningl2  - can you ask Ingrid to add the landing page today.

Reply:
If it's not possible to add the landing page as per John's suggestion, line 77 can instead point to: src/THEOplayer/README.md

If you want me to do it and resubmit, let me know.